doc_template/visual_behavior_ophys_nwb.rst

```diff
@@ -0,0 +1,110 @@
+The contents of a 'file' within HDF5, which called a dataset, can be seen by double-clicking it. 
+An NWB file has seven top-level folders and five top-level datasets. 
+In Figure 1, the contents of the 'session_description' dataset is displayed.
+An NWB file is designed to store data from a single experimental session on a single animal. 
+The `session_description` dataset provides a quick summary of the information available in the file.
+The NWB specification, describing the format and how data is stored, can be downloaded `here <http://github.com/NeurodataWithoutBorders/specification/blob/master/version_1.0.3/nwb_file_format_specification_v1.0.3.pdf>`_.
+
+
+Stimulus data
+-------------
+
+Within an NWB file, stimulus data is organized into 'templates', which are stimulus descriptions that can be used one or more times, and 'presentation', which stores the time-series data about what was presented and when. 
+
+.. figure:: /_static/behavior_ophys_nwb_fig-2.png
+
+   **Figure 2. Stimuli presented during the experiment**
+
+In this experiment shown in Figure 2, there was only one stimulus presented - a set of natural images. 
+When browsing the file, note the panel to the right. 
+Clicking on a folder or dataset will cause information about that object to be displayed there. 
+This will display a table of information related to that dataset, including comments, descriptions, and other relevant metadata. 
+Double clicking one of these will further reveal the value stored inside.
+
+Processing
+----------
+
+In the NWB file, the processing folder is designed to store the contents of the different levels of intermediate processing of data that are necessary to convert raw data into something that can be used for scientific analysis, as well as information about the behavior of the specimen during data collection. 
+In the Allen Institute's NWB file, these levels of processing include motion correction of the acquired 2-photon movie frames, image segmentation into regions of interest, the fluorescence signal for each of these regions, and the dF/F signal that is useful for correlating cell activity with presented stimuli. 
+
+This is also where you will find the main difference between the two types on NWB files produced for the Allen Institute Brain Observatory dataset. 
+In the processing directory, you will notice behavior data information like licking, running, rewards, etc. 
+If this is an NWB strictly containing behavior data, then that will be all. If the NWB file is one that includes optical physiology data as well then there wil also be a directory in processing called `ophys`, where you will find the intermediate processing data. 
+
+.. figure:: /_static/behavior_ophys_nwb_fig-3.png
+
+   **Figure 3. Location of dF/F data**
+
+
+HDFView provides a very efficient way to browse the contents of an NWB file.
+
+Below is a brief description and location for several types of data in the NWB file.
```

Comment:
The table below seems to missing a significant amount information, did you forget to commit a file?

Reply:
The table was never meant to contain all of the information, it says "location for several types of data." Should we add the whole thing?

Reply:
I guess I wonder what's the point of the table if it is nowhere near complete? Maybe consult with scientific stakeholders and see what they want?

Reply:
I'm not really sure, it's just what was in the original NWB page. Probably just as a sample of the kinds of things that were in it. But I do agree that a complete table would be much better. I'll reach out on slack or more info

Reply:
Let's fill this thing out.  We can start here so that we can add and edit collaboratively, and get Marina and Co to review and update, and then update the code: http://confluence.corp.alleninstitute.org/display/IT/Visual+Behavior+NWB+data+fields